src/components/footer/footer.scss

```diff
@@ -96,11 +94,11 @@ on tablets. Can be updated per UX team as necessary.
         ul {
             list-style: none;
             text-transform: uppercase;
-            padding-left: 3.1rem;
-            @media (max-width: 766px) {
-                padding-left: 0;
-                padding-top: 1.5rem;
-                text-align: center;
+            padding: 1.5rem auto auto 0;
```

Comment:
This isn't valid styling, and honestly probably isn't needed. Play around with it a bit in dev tools to make sure. When you spin up your dev server and take a look at the mobile footer, this styling is currently being ignored. You can tell because it is crossed out when you inspect the element and view the styles. It also has a yellow triangle with an exclamation point next to it which is an indicator that it is not be applied, not because it is overwritten, but because the search engine doesn't know what to do with it.

Reply:
Thanks for the tip about the yellow triangle! And yah, the original padding-top is so minuscule, I eliminated it. Thanks!